.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+jobs:
+    build:
+        docker:
+            - image: circleci/python:3.6.1
+    steps:
+        - checkout
+    run:
+        command: |
+          wget -q http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O ~/miniconda.sh;
+          chmod +x ~/miniconda.sh;
+          ~/miniconda.sh -b -p ~/miniconda;
+          echo "export PATH=~/miniconda/bin:$PATH" >> $BASH_ENV;
+    run:
+        command: |
+          conda update --yes --quiet conda;
+          conda env create -n testenv --yes pip python=3
+          source activate testenv;
+          conda install --yes --quiet numpy=1.13.1 scipy=0.19.1 matplotlib pandas=0.20.3
+          pip install flake8 pytest nose coverage coveralls
+          git clone --depth 1 https://github.com/mne-tools/mne-python.git -b maint/0.15
+          cd mne-python
```

Comment:
Cd mne_bids

Reply:
actually, that part is for installing mne-python. mne-bids is installed a line lower using `python setup.py develop`

Reply:
I would do it with

pip install -U https://api.github.com/repos/mne-tools/mne-python/zipball/maint/0.15

I should work.

Reply:
The issue is that we use the data in `mne/tests/data` for our testing which isn't downloaded with `pip`. You need it there. But for CircleCI I think a normal pip install should do the job